diff --git a/objectsharer/objectsharer.py b/objectsharer/objectsharer.py
index 8398ccb..e18e619 100644
--- a/objectsharer/objectsharer.py
+++ b/objectsharer/objectsharer.py
@@ -223,7 +223,7 @@ class ObjectSharer(object):
         self._last_call_id = 0
         self.reply_objects = {}
 
-        # Signal related 
+        # Signal related
         self._last_hid = 0
         self._callbacks_hid = {}
         self._callbacks_name = {}
@@ -570,6 +570,8 @@ class ObjectSharer(object):
 
         if info[0] == OS_CALL:
             # In a try statement to postpone checks
+            obj = None
+            func = None
             try:
                 (callid, objid, funcname, args, kwargs) = info[1:6]
                 sig = kwargs.get(OS_SIGNAL, False)
